packages/manager/modules/pci/src/projects/project/instances/add/translations/Messages_fr_FR.json

```diff
@@ -8,7 +7,7 @@
   "pci_projects_project_instances_add_region_datacenter_title": "Région",
   "pci_project_instances_instance_add_region_error": "Une erreur est survenue lors de la récupération des régions disponibles",
   "pci_project_instances_instance_add_region_activate": "Activer cette région",
-  "pci_project_instances_instance_add_region_quota": "Augmenter mon quota",
+  "pci_project_instances_instance_add_region_quota": "Le quota de votre projet ne vous permet pas d’ajouter ce type de machine dans cette région. <a class=\"oui-link\" href=\"{{ quotaUrl }}\">Gérer mon quota</a>",
```

Comment:
Can we split the translation into multiple chunks? (_as I mentioned in a previous comment_).